tutorials/tutorial1.ipynb

```diff
@@ -10,11 +10,11 @@
     "\n",
     "In this tutorial we train an AtacWorks model to denoise the signal track and call peaks from aggregate single-cell ATAC-seq data derived from a small number of cells. We use the dsc-ATAC-seq dataset presented in reference (1) (Section \"AtacWorks enhances ATAC-seq results from small numbers of single cells\", also Supplementary Table 8). This dataset consists of single-cell ATAC-seq data from several types of human blood cells.\n",
     "\n",
-    "Note that all the AtacWorks models described in reference (1) are available to download (https://atacworks-paper.s3.us-east-2.amazonaws.com) and you may be able to use one of these instead of training a new model. To learn how to download and use an existing model, refer to [Tutorial 2](tutorial2.ipynb).\n",
+    "Note that all the AtacWorks models described in reference (1) are available to download (https://ngc.nvidia.com/catalog/models). Using the search bar in the NGC catalog, type \"atac\" for all the AtacWorks pre-trained models to be listed. You may be able to use one of these instead of training a new model. To learn how to download and use an existing model, refer to [Tutorial 2](tutorial2.ipynb).\n",
     " \n",
-    "We selected 2400 Monocytes from this dataset - this is our ‘clean’, high-coverage dataset. We then randomly sampled 50 of these 2400 Monocytes. Here's what the ATAC-seq signal from 50 cells and 2400 cells looks like, for a region on chromosome 10:\n",
+    "In this tutorial, we selected 2400 Monocytes from a dataset - this is our ‘clean’, high-coverage dataset. We then randomly sampled 50 of these 2400 Monocytes. Here's what the ATAC-seq signal from 50 cells and 2400 cells looks like, for a region on chromosome 10:\n",
     "\n",
-    "![Monocytes subsampled signal](../docs/tutorials/Mono.2400.50.png)\n",
```

Comment:
this image doesn't appear in the file.

Reply:
this is old log. It will get updated to 10 epochs after I run again